pkg/istio-agent/sds-agent.go

```diff
@@ -127,6 +128,10 @@ const (
 	initialBackoffInMilliSec = "INITIAL_BACKOFF_MSEC"
 
 	pkcs8Key = "PKCS8_KEY"
+
+	// The environmental variable name to specify that private keys should be
+	// generated using Elliptical Curve Cryptography
+	eccKey = "ECC_KEY"
```

Comment:
Since there are many types of ECC keys - how about having this as a String instead of bool, with the value indicating the curve ? 

Also - we generally want to use ProxyConfig for all proxy-related configuration instead of env. We still use env - hard to migrate, and some things are just experimental - but this seems like a long-lived feature.

Reply:
I'm ok with starting as an env - but at least let's have a more future-proof setting.